deployment/common/changeset/test_helpers.go

```diff
@@ -48,6 +51,23 @@ func ApplyChangesets(t *testing.T, e deployment.Environment, timelockContractsPe
 		} else {
 			addresses = currentEnv.ExistingAddresses
 		}
+		if out.JobSpecs != nil {
```

Comment:
Here, you might consider deleting this, or throwing a warning for any tests that continue to return job-specs. The issue is not this code, it was the removal of the field from the type, which makes the code a breaking change in CLD. This test code, if you've eliminated usage, can go, or be altered to panic to make sure.

Reply:
Not necessary, just a thought, to avoid folks adding usages in chainlink/deployments before you get to delete this.